### ISSUE TITLE
About CrossEntroyLoss

### ISSUE THREAD

**Issue by SmartAI** [vinhkhuc/PyTorch-Mini-Tutorials#1]
The pytorch document shows:
```
class torch.nn.CrossEntropyLoss(weight=None, size_average=True)[source]
This criterion combines LogSoftMax and NLLLoss in one single class.
```
SO could I remove `softmax` layer from the model sequential?
```
model.add_module("softmax", torch.nn.Softmax())
```

**Comment by vinhkhuc** [comment]
I've checked CrossEntropyLoss and found that it computes softmax internally. So, the softmax layer is unnecessary. Thanks for pointing that out.

**Comment by brando90** [comment]
do you have a line reference for this in the pytorch documentation?
